### PR TITLE
First draft of Sydney cuisine and activity description document.

### LINKED ISSUE #11: delilah-cisra recommendations for Sydney
There are many things one can do in Sydney for fun and profit:
Macquarie Park is of course a well-known tourist mecca; and
Newtown and Crows Nest are good places to go for food.

### PR DESCRIPTION
**Pull request by delilah-cisra**
This closes #11, mostly:-)

Adding a file with Sydney-centric recommendations.

@nikosandronikos Do you think I should add a mention for Los Vida?:-)

### CODE REVIEW

**Comment by delilah-cisra** [comment]
@sammo2828 please let me know if you would like any Mexican restaurants added to this document.

File: delilah-cisra-sydney.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Sydney
+
+## Places to eat
```
Comment (sammo2828) [inline delilah-cisra-sydney.md:3]: You might want to use ```###``` subheading for this

**Comment by sammo2828** [comment]
There is a nice Mexican restaurant called Los Vida in Crows Nest :taco:

**Review by sammo2828 — APPROVED** [review]
I'm happy with your changes. Go ahead and merge.

**Pull request merged**